mojo/core/broker_castanets.h

```diff
@@ -127,6 +130,8 @@ class BrokerCastanets : public Channel::Delegate, public base::SyncDelegate {
 
   std::unique_ptr<CastanetsFenceQueue> fence_queue_;
 
+  base::OnceCallback<void()> tcp_success_callback_;
```

Comment:
Both Nodecontroller and BrokerCastanets has a callback for TCP connection success?

Reply:
This is just a residue in my previous patchset.
Let me remove this.